integrations/docker/images/stage-3/chip-build-android/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG VERSION=1
+ARG VERSION=2
```

Comment:
same here: this line should not change, just change the global one.

Reply:
Thank you. Modified.